conf/tests/test_GRCh37.config

```diff
@@ -13,11 +13,7 @@
 params {
     config_profile_name        = 'Test profile'
     config_profile_description = 'Minimal test dataset to check pipeline function and save GRCh37 references'
-    input = "${projectDir}/tests/data/samplesheet_WES_test.csv"
```

Comment:
I would prefer to keep this as "input" since it is conventional with other nexflow pipelines. --input is alwasy samplesheet in nextflow.